api/R/insert_new_workflow.R

```diff
@@ -44,7 +44,7 @@ insert_new_workflow <- function(con,
     length(user_id) <= 1,
     length(con) == 1,
     # Must be RPostgres connection for prepared queries 
-    class(con) == "PqConnection"
+    is.PqConnection(con)
```

Comment:
There is no such function, `is.PqConnection` (though there probably should be in the `RPostgres` package!). We could implement it ourselves, but it's probably easier to just do `inherits(con, "PqConnection")`.